### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by oblm**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Changes made to improve the clarity and accuracy of the CHANGELOG.

#### Release note
This PR is nothing but release notes.

#### Tests
Rendered markdown.

#### Documentation
This PR is nothing but documentation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,27 +4,43 @@ All notable changes to the SpatialOS Game Development Kit for Unreal will be doc
...
-- Fixed a bug that could caused a name collision in schema for sublevels.
+- Fixed a bug that could cause name collisions in schema generated for sublevels.
 - Downgraded name collisions during schema generation from Warning to Display.
 - Replicating a static subobject after it has been deleted on a client no longer results in client attaching a new dynamic subobject.
 - Fixed a bug that caused entity pool reservations to cease after a request times out.
 - Running `BuildWorker.bat` for `SimulatedPlayer` no longer fails if the project path has a space in it.
 - Fixed a crash when starting PIE with out-of-date schema.
 
+### Internal:
+Features listed in this internal section are not ready for you to use but, in the spirint of open development, we detail every change we make to the GDK.
```
Comment (improbable-valentyn) [inline CHANGELOG.md]:
```suggestion
Features listed in this internal section are not ready for you to use but, in the spirit of open development, we detail every change we make to the GDK.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,27 +4,43 @@ All notable changes to the SpatialOS Game Development Kit for Unreal will be doc
...
-- A configurable actor component 'SpatialPingComponent' is now available for player controllers to measure round-trip ping to their current authoritative server worker. The latest ping value can be accessed raw through the component via 'GetPing()' or otherwise via the rolling average stored in 'PlayerState'.
-- Added the AllowUnresolvedParameters function flag that disables warnings for processing RPCs with unresolved parameters. This flag can be enabled through Blueprints or by adding a tag to the `UFUNCTION` macro.
+- You can now call `SpatialToggleMetricsDisplay` from the console in your Unreal clients in order to view metrics. `bEnableMetricsDisplay` must be enabled on clients where you want to use this feature.
+- The modular-udp networking stack now uses compression by default.
+- Reduced network latency by switching off default rpc-packing. If you need this on by default, you can re-enable it by editing `SpatialGDKSettings.ini`
+- When you start a local deployment the GDK now checks the port required by the runtime and, if its in use, prompts you to kill that process.
```
Comment (improbable-valentyn) [inline CHANGELOG.md]:
```suggestion
- When you start a local deployment, the GDK now checks the port required by the runtime and, if it's in use, prompts you to kill that process.
```

File: CHANGELOG.md (hunk 1 above)
Comment (oblm) [reply CHANGELOG.md]: haha, JIRA on the brain, thanks

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,27 +4,43 @@ All notable changes to the SpatialOS Game Development Kit for Unreal will be doc
...
-- Fixed a bug that could caused a name collision in schema for sublevels.
+- Fixed a bug that could cause name collisions in schema generated for sublevels.
 - Downgraded name collisions during schema generation from Warning to Display.
 - Replicating a static subobject after it has been deleted on a client no longer results in client attaching a new dynamic subobject.
 - Fixed a bug that caused entity pool reservations to cease after a request times out.
 - Running `BuildWorker.bat` for `SimulatedPlayer` no longer fails if the project path has a space in it.
 - Fixed a crash when starting PIE with out-of-date schema.
 
+### Internal:
+Features listed in this internal section are not ready for you to use but, in the spirit of open development, we detail every change we make to the GDK.
```
Comment (improbable-valentyn) [inline CHANGELOG.md]:
```suggestion
Features listed in this internal section are not ready to be used but, in the spirit of open development, we detail every change we make to the GDK.
```

**Pull request merged**